lib/applicaster/test/accounts/web_mock_helper.rb

```diff
@@ -9,6 +9,14 @@ module Applicaster
   module Test
     module Accounts
       module WebMockHelper
+        def stub_user_show_response(options = {})
```

Comment:
since this is meant to be merged in someone else's code base maybe it's better to name it `stub_accounts_user_show_response` or `stub_accounts_get_user_response`

It could be a good idea to follow rails's route helpers conventions to name these:
`users#show` -> `user_path` -> `stub_accounts_user`
`users#current` -> `current_user_path` -> `stub_accounts_current_user`

just a thought...